Plash/AppDelegate.swift

```diff
@@ -443,4 +445,30 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 
 		menu.addQuitItem()
 	}
+
+	/**
+	Scans the given URL for placeholders and subtitutes them with
+	their corresponding values.
+	*/
+	private func replacePlaceholders(of url: URL) -> URL? {
+		// Here we swap out [[screenWidth]] and [[screenHeight]] for their actual values.
+		// We proceed only if we have an NSScreen to work with.
+		guard let screen = desktopWindow.targetScreen?.withFallbackToMain ?? .main else {
+			print("No screen was found to read dimensions from!")
```

Comment:
No need to print. This can just fail silently.

Reply:
👍